src/pytesmo/metrics/confidence_intervals.py

```diff
@@ -328,15 +328,15 @@ def _jackknife(metric_func, x, y):
 
 def _percentile_bs_ci(bs_m, m, alpha):
     """Calculates the CI using percentiles"""
-    lower = np.quantile(bs_m, alpha / 2)
-    upper = np.quantile(bs_m, 1 - alpha / 2)
+    lower = np.nanquantile(bs_m, alpha / 2)
```

Comment:
is it expected that there can be nans when using small samples?

Reply:
I think that the metric functions should normally not return NaN, but it's probably safer to use nanquantile